### DISCUSSION TITLE
GNN-powered Classification and Explanation App

### DISCUSSION THREAD

**Discussion by fork123aniket** [pyg-team/pytorch_geometric#7935] [Show and tell]
Have just deployed a `GNN-based Node and Graph Classification and Explanation App` using `Streamlit`, more information on which can be found [***here***](https://github.com/fork123aniket/End-to-End-Node-and-Graph-Classification-and-Explanation-App). The deployed app can be accessed through this [***link***](https://graphexplanability.streamlit.app). My question: is it possible to include this as an example of `Graph Explainability` under `torch_geometric.explain` section on `PyG official docs`?? Note that have used `GraphMaskExplainer` of `PyG` for `Node` and `Graph Classification` tasks.

**Comment by rusty1s** [comment]
Very cool work. We could list this in our external resources page: https://pytorch-geometric.readthedocs.io/en/latest/external/resources.html. I am also open to adding a link to this in the `GraphMaskExplainer` documentation.
